raml-parser-interface/src/main/java/org/mule/apikit/model/parameter/Parameter.java

```diff
@@ -18,6 +19,15 @@ public interface Parameter {
 
   String getDefaultValue();
```

Comment:
We should do something like this so our tools help us diagnose any missing changes :P
```suggestion
  /**
   * @deprecated Please use {@link getDefaultValues} instead
   */
  @Deprecated
  String getDefaultValue();
```

additionally it would be nice to have something like this
```java
Optional<String> tryGetDefaultValue() default {
  return getDefaultValues().stream().findFirst();
}
```

Reply:
At least that's my understanding of the changes (we are changing getDefaultValue() with getDefaultValues() everywhere).

Reply:
Yeah, agree, thanks for recall me that, I was going to ask for you opinion about it. I had added the annotation and then regretted because I wasn't sure if it would be useful.
I understand that adding the default method that you are suggesting will be helpful for those clients that are pretty sure there is only one value, but for the time being, imho, I would avoid increasing the api with a helper method. What do you think about it @JuanAller @juanchib?

Reply:
Agree that adding a default might be a waste of time :P